staff/marc-uson/spotify-template/style/aspect.sass

```diff
@@ -0,0 +1,26 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat')
```

Comment:
@Markuson nice use of variables! (usually this file is named variables.sass)

Reply:
I called it aspect cause it refers to aspect variables.

maybe better call it "aspect-variables.sass"?

Reply:
as you wish!